opentech/static_src/src/javascript/apply/toggle-proposal-info.js

```diff
@@ -0,0 +1,24 @@
+(function ($) {
+
+    'use strict';
+
+    function toggleProposalInfo() {
+        $('.js-toggle-propsoal-info').click(function (e) {
+            e.preventDefault();
+            const activeClass = 'is-open';
+
+            if (this.innerHTML === 'Show more') {
```

Comment:
@chris-lawton _Could_ be neater like this?... but up to you, naturally (might not even work 😅 🤷‍♂) 

```
const innerHtml = this.innerHTML;
const showMore = 'Show more';
innerHtml = (innerHtml === showMore) ? 'Hide' : showMore;
```